packages/extension-ui/src/components/Address.tsx

```diff
@@ -35,7 +35,7 @@ interface Props {
   genesisHash?: string | null;
   isExternal?: boolean | null;
   isHidden?: boolean;
-  name?: React.ReactNode | null;
```

Comment:
Not sure why this was the a node, it didn't sound sane, and didn't break anything (TS would tell us)

Reply:
Probably a catch-all. But since it is quite controlled, shouldn’t make a difference here.

Reply:
right, since I use it below as a string, and TS was complaining that I couldn't use a node, shrinking this down helped me a little.